docs/howto/error-reporting.txt

```diff
@@ -325,6 +325,16 @@ Your custom reporter class needs to inherit from
 
 .. class:: ExceptionReporter
 
+    .. attribute:: html_template_path
+
+        A :class:`pathlib.Path` representing the filesystem path to a template
+        for rendering the HTML representation of the exception.
+
+    .. attribute:: text_template_path
+
+        A :class:`pathlib.Path` representing the filesystem path to a template
+        for rendering the plain-text representation of the exception.
+
```

Comment:
Hey @akx @pope1ni. 

* I think we should say `absolute filesystem path` here. (Because we can't use the template in-case the error is there — which we don't need to say here.) 
* **Question**: do you think we should say `Defaults to the Django provided template.` or similar? (i.e. is it obvious enough that I don't need to set this unless I want to?)

Thanks